pkgs/servers/mail/exim/default.nix

```
@@ -92,11 +92,11 @@ stdenv.mkDerivation rec {
       done )
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     homepage = "http://exim.org/";
     description = "A mail transfer agent (MTA)";
-    license = stdenv.lib.licenses.gpl3;
-    platforms = stdenv.lib.platforms.linux;
-    maintainers = [ stdenv.lib.maintainers.tv ];
+    license = with licenses; [ gpl3 ];
+    platforms = with platforms; [ linux ];
```

This disables the derivation on all platforms ;).

```suggestion
    platforms = with platforms; linux;
```